src/test/java/emissary/test/core/junit5/ExtractionTest.java

```diff
@@ -218,6 +218,11 @@ protected void checkAnswers(Element el, IBaseDataObject payload, List<IBaseDataO
             assertEquals(shortName, payload.shortName(), "Shortname does not match expected in " + tname);
         }
 
+        String fontEncoding = el.getChildTextTrim("fontEncoding");
+        if (fontEncoding != null && fontEncoding.length() > 0) {
```

Comment:
```suggestion
        if (StringUtils.isNotBlank(fontEncoding)) {
```
Or did I miss something?

Reply:
Everything else in this PR looks GTG